### DISCUSSION TITLE
This run was triggered by a pull request and therefore a new version won't be published.

### DISCUSSION THREAD

**Discussion by natanans** [semantic-release/semantic-release#2795] [Q&A]
So the above warning comes up on a new repo that I have setup after merging a pull request...but the thing is on other repos a merge to a release branch would trigger a release...and all repos are using the same .releaserc...can any one help?

**Comment by travi** [comment]
we need more detail to consider providing help. semantic-release needs to run on a branch build rather than a pull request build. you mention this happening _after_ merging a pull request. are you suggesting that a branch build is incorrectly being detected as a pull request build? do you have a public reproduction that you could link to?
